src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -353,97 +353,57 @@ export class StudyViewPageStore {
             onMobxPromise(this.selectedSamples,
                 async (selectedSamples)=>{
                     // get clinical data for the given attribute
-                    const entityIdKey = (clinicalAttribute.patientAttribute ? "patientId" : "sampleId")
-                    const data = await client.fetchClinicalDataUsingPOST({
+                    const entityIdKey = (clinicalAttribute.patientAttribute ? "patientId" : "sampleId");
+                    let data = await client.fetchClinicalDataUsingPOST({
                         clinicalDataType: clinicalAttribute.patientAttribute ? "PATIENT" : "SAMPLE",
                         clinicalDataMultiStudyFilter: {
                             attributeIds: [clinicalAttribute.clinicalAttributeId],
                             identifiers: selectedSamples.map(s=>({ studyId: s.studyId, entityId: s[entityIdKey] }))
                         }
                     });
 
-                    // sort values
-                    // group into halves, thirds, or fourths, depending on how many distinct values there are
-                    const numericalData = data.filter(d=>!isNaN(d.value as any));
-                    const groupedByValue = _.groupBy(numericalData, d=>parseFloat(d.value));
-                    const distinctValues = _.sortBy(Object.keys(groupedByValue).map(v=>parseFloat(v)));
-
-                    statusCallback(LoadingPhase.CREATING_SESSION);
+                    // get quartiles of data
```

Comment:
we should really have a test around this stuff, no?   we could follow up with that